src/helpers/spatialHelper.ts

```diff
@@ -1,5 +1,17 @@
-import type { Bbox } from "../types";
+import type { Bbox, GeographyLookupProps } from "../types";
 
 export const getBboxString = (args: Bbox) => {
   return [args.east, args.north, args.west, args.south].map((n) => n.toFixed(6)).join(",");
 };
+
+export const defaultGeography: GeographyLookupProps = {
+  meta: {
+    name: "England and Wales",
+    code: "K04000001",
+    geotype: "EW",
```

Comment:
reviewers - should this be `ew` not `EW`?

Reply:
(the API response will have it as EW, but it means this object isn't as useful as a general source-of-truth as it could be...)

Reply:
I'm sure you're right - the type in types.ts has it as `ew`